src/main/java/com/google/api/gax/grpc/BundlingCallSettings.java

```diff
@@ -0,0 +1,111 @@
+package com.google.api.gax.grpc;
+
+import com.google.api.gax.core.RetrySettings;
+
+import io.grpc.MethodDescriptor;
+import io.grpc.Status;
+
+import java.io.IOException;
+import java.util.Set;
+
+/**
+ * A settings class with generic typing which can be used to configure a bundling method
```

Comment:
Remove the part about 'generic typing' on all of the concrete call settings classes.